.qi.yml

```diff
@@ -1,20 +1,18 @@
 # A configuration file for our Quality Infrastructure builds: https://wiki.gpii.net/w/Quality_Infrastructure
-app_name: gpii-pouchdb
-
-git_repository: https://github.com/the-t-in-rtf/gpii-pouchdb.git
-
 email: tony@raisingthefloor.org
-
-env_runtime: linux
-
-app_start_service: false
-
-software_stack: nodejs
-
-software_stack_version: lts
-
-setup:
-  - npm install
-
-commands:
-  - npm test
+env_runtime: linux-desktop
+
+apps:
+  - app_name: gpii-pouchdb
+    git_repository: https://github.com/the-t-in-rtf/gpii-pouchdb.git
+    git_branch: GPII-1815
```

Comment:
This line is even more worrying now it no longer even refers to the correct branch. Shall we just delete this file entirely for now?